module/config.mjs

```diff
@@ -841,6 +841,37 @@ preLocalize("featureTypes.class.subtypes", { sort: true });
 
 /* -------------------------------------------- */
 
+/**
+ * Configuration data for an item with the "loot" type.
+ *
+ * @typedef {object} LootTypeConfiguration
+ * @property {string} label                       Localized label for this type.
+ */
+
+/**
+ * Types of "loot" items.
+ * @enum {LootTypeConfiguration}
+ */
+DND5E.lootTypes = {
+  junk: {
+    label: "DND5E.Loot.Junk"
+  },
+  resource: {
+    label: "DND5E.Loot.Resource"
+  },
+  treasure: {
+    label: "DND5E.Loot.Treasure",
+    subtypes: {
```

Comment:
We have `loot` which is a sub-type of Item, and `treasure` which is a sub-type of `loot`, and now `gem` which is a sub-type of `treasure`. I think we could simplify things in a few places if we flattened this out and had `gem` and `art` just be direct sub-types of `loot`.

At most we could group them in an `<optgroup>` like we do for armour.

![image](https://github.com/foundryvtt/dnd5e/assets/207433/1a14f17c-66fd-4c4e-99a2-34a2af4261a9)